common/rnc_deco.h

```diff
@@ -19,14 +19,11 @@
  *
  */
 
-#ifndef SKY_RNC_DECO_H
-#define SKY_RNC_DECO_H
+#ifndef COMMON_RNC_DECO_H
+#define COMMON_RNC_DECO_H
 
 
-
-#define RNC_SIGNATURE   0x524E4301 // "RNC\001"
-
-namespace Sky {
+namespace Common {
 
 class RncDecoder {
```

Comment:
Since this will be moved under `common`, please provide a comment with a description about this class

Reply:
Done

Reply:
@phcoder This one is not resolved. Please recheck

Reply:
Sorry, replied before I did the push. Sorry for race condition